.gitignore

```diff
@@ -39,7 +39,6 @@
 #
 ################################################
 
-config/local.js
```

Comment:
This seems useful to keep locally, for local only settings. I wasn't sure if you were using development for that purpose-- if so we can get rid of this.

Reply:
imo local only => different on each computer. Shared settings that are the same for localhost development belong in development.js